src/tests/AddRepoForm.test.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import "@testing-library/jest-dom/extend-expect";
-import {render, cleanup} from "@testing-library/react";
-import AddRepoForm from "../components/AddRepoForm";
+import {render, cleanup, fireEvent} from "@testing-library/react";
```

Comment:
fireEvent is unused, can you remove

Reply:
restore ```AddRepoForm.test.js```` as moved all test cases to ```tests/util.test.js```. Hence it is not required any more.